brainbox/examples/docs_load_spike_sorting.py

```diff
@@ -37,14 +37,7 @@
 #  'spikes.times']
 # If we also want to load for example, 'clusters.peakToTrough we can add a dataset_types argument
 
-spikes, clusters, channels = bbone.load_spike_sorting_with_channel(eid, one=one, probe='probe00')
-print(clusters['probe00'].keys())
-
-##################################################################################################
-# Example 4:
-# By default, load_spike_sorting_with_channel will look for available data on your
-# local computer and if all the datasets are found it will not check for consistency with files
-# stored on flatiron (to speed up loading process). To make sure that data is always synced to
-# flatiron we can set a force flag to True
-spikes, clusters, channels = bbone.load_spike_sorting_with_channel(eid, one=one, probe='probe00')
+spikes, clusters, channels = bbone.load_spike_sorting_with_channel(eid, one=one, probe='probe01',
+                                                                   dataset_types=['clusters.peakToTrough'])

```

Comment:
I was a bit cavalier in refactoring `load_spike_sorting` and removed the dataset_types kwarg. If you think it's worth keeping perhaps rename it to `datasets` instead.  At the moment it will download the full spikes object.  For now you could use `|` in attribute to load selected attributes (I still haven't updated it to support lists):
```
default_attr = [
    'times',
    'amplitudes'
]
spikes = one.load_object(eid, 'spikes', attribute='|'.join(default_attr), collection=f'alf/{probe_label}')
```

Reply:
This one, I have already changed so it maintains the same behaviour as the old code. I could be inclined to keep the argument as dataset_types just to causes as little disruption to user code as possible. But also happy to change if you have strong opinions otherwise :)